### ISSUE TITLE
Dimensions of M and K

### ISSUE THREAD

**Issue by qiongsiwu** [dilevin/CSC2549-a2-mass-spring-3d#7]
This is related to issue #5. 

Lecture 3 slide on Kinetic Energy indicates that the dimension of the mass matrix M is nxn. I am assuming the "n" here means the number of vertices, because on the slide on generalized coordinates, we say q is a point in R^(3n). However, we have qT M q in the computation of kinetic energy, and to match the dimension, M should have dimension 3n x 3n. 

In the implicit Euler integration formula, we have (M - dt^2 * K), which implies M and K should have the same dimension, which implies K should have dimension 3n as well. 

In summary, may I get some clarification on the meaning of 

- n

- m : I believe this means the number of springs. 

- dimension of M

- dimension of K

Thanks so much!

**Comment by dilevin** [comment]
In the lectures and notes in will try to consistnetly use n to be the number of vertices in the mesh and m to be the number of elements (in thus case, springs). With these definitions we have: 

q is 3nx1
M and K are 3n x 3n

**Comment by dilevin** [comment]
to unify the discussion of fixed vertices. If c is the number of constrained vertices then P is a 3(n-c)x3n matrix
